### PR TITLE
Pet Skins and Ears

### PR DESCRIPTION
**Pull request by SabreCat**
Implements the following:
- A new set of always-available premium avatar skins based on eight of the basic pets.
- A new set of head accessories matched to those skins.
- The ability to display and purchase a set of gear items on the Avatar Customization page

In support of the above, Lodash was added to `middleware.js`.

### CODE REVIEW

File: website/src/middleware.js [hunk 1]
```diff
@@ -198,7 +198,8 @@ module.exports.locals = function(req, res, next) {
     Content: shared.content,
     mods: require('./models/user').mods,
     tavern: tavern, // for world boss
-    worldDmg: (tavern && tavern.quest && tavern.quest.extra && tavern.quest.extra.worldDmg) || {}
+    worldDmg: (tavern && tavern.quest && tavern.quest.extra && tavern.quest.extra.worldDmg) || {},
+    _: _
```
Comment (crookedneighbor) [inline website/src/middleware.js:202]: Is this to provide lodash as a resource to something on the server? What?

Comment (SabreCat) [inline website/src/middleware.js:202]: It's for this: https://github.com/SabreCat/habitrpg/blob/pet-skins/website/views/options/profile.jade#L7

That's what @lefnire and I came up with after banging around quite a bit, but if there's a smoother way to get this logic in without needing lodash as middleware, that may be preferable.

**Comment by hairlessbear** [comment]
@SabreCat This is incredibly minor, but is there any chance that you could add a credit to me in the Bailey announcement for the Inn improvements? I submitted the [Inn behavior change PR](https://github.com/HabitRPG/habitrpg/pull/5073) and collaborated with Blade on it.

**Comment by crookedneighbor** [comment]
@SabreCat Yes, definitely do that.

**Comment by SabreCat** [comment]
Aha! Absolutely, thanks.

**Pull request merged**